packages/notification/channel/webhook.ts

```diff
@@ -36,12 +36,24 @@ export const validator = Joi.object().keys({
 
 export const send = async (
   { url }: NotificationData,
-  { body }: NotificationMessage
+  { meta }: NotificationMessage
 ): Promise<void> => {
+  if (meta.type !== 'incident' && meta.type !== 'recovery') {
```

Comment:
why add this condition? how about the start messsage with `meta.type === 'start'`?

Reply:
I assume that the webhook notification is created when there is only incident type. Because, if you read [the documentation](https://monika.hyperjump.tech/guides/notifications#webhook), it only sends the following data:

```json
{
 "body": {
   "url": "string"
   "time": "string"
   "alert": "string"
  }
}
```

Only notification with incident or recovery types that have url, time, and alert data. So, I added the condition. Otherwise, it will send empty data.

Of course, we need to improve the webhook data in the future to send other type of notification. But this PR only fixes the existing feature.

Reply:
i think one of the most basic and important is the `start notification` to know that the notification is working. It should be next in line